raiden/storage/serialization/types.py

```diff
@@ -84,7 +84,7 @@
 
 MESSAGE_NAME_TO_QUALIFIED_NAME = {
     "AuthenticatedMessage": "raiden.messages.abstract.AuthenticatedMessage",
-    "Claim": "raiden.claim.Claim",
+    "Claim": "raiden.transfer.state.Claim",
```

Comment:
I wonder why we did this in the first place. The claim is not a message in the traditional sense.

Reply:
Why have it in there? 
Because this is used by the (De-)Serializing machinery to find the correct class. 
I guess we could also use the DictSerializer directly and just pass the correct class path...

Reply:
Sure, but I think we never explicitly enumerate all state classes.